lib/index.js

```diff
@@ -16,7 +16,18 @@ exports.Upload = require('./Upload')
  * @prop {string} filename File name.
  * @prop {string} mimetype File MIME type. Provided by the client and can’t be trusted.
  * @prop {string} encoding File stream transfer encoding.
- * @prop {Function} createReadStream Returns a Node.js readable stream of the file contents, for processing and storing the file. Multiple calls create independent streams. Throws if called after all resolvers have resolved, or after an error has interrupted the request.
+ * @prop {FileUploadCreateReadStream} createReadStream Creates a [Node.js readable stream](https://nodejs.org/api/stream.html#stream_readable_streams) of the file’s contents, for processing and storage.
+ */
+
+/**
+ * Creates a [Node.js readable stream](https://nodejs.org/api/stream.html#stream_readable_streams) of an [uploading file’s]{@link FileUpload} contents, for processing and storage. Multiple calls create independent streams. Throws if called after all resolvers have resolved, or after an error has interrupted the request.
+ * @kind typedef
+ * @name FileUploadCreateReadStream
+ * @type {Function}
+ * @param {object} [options] [`ReadStreamOptions`](https://github.com/mike-marcacci/fs-capacitor#readstreamoptions) for `fs-capacitor`.
+ * @param {string} [options.encoding] Optional value to use as the readable stream's highWaterMark, specifying the number of bytes (for binary data) or characters (for strings) that will be bufferred into memory. [See node's docs for `stream.Readable`.](https://nodejs.org/api/stream.html#stream_new_stream_readable_options)
```

Comment:
- No need to say "Optional value" as that is documented via JSDoc.
- This option is `encoding`; this description is incorrect as it's a copy of the `highWaterMark` description.
- The `highWaterMark` (although it should be `encoding`) should be backticked for display as inline code.
- "bufferred" is incorrect spelling.
- Regarding `See node's docs`:
  - `node's` should be `Node.js`, we display the proper name everywhere.
  - This should be linked using a propper `@see` JSDoc tag.

Also, it might be useful to hint about what these options might be useful for.